app/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: collection
 title: Becoming a teacher design history
-description: A history of the designs for the Find, Apply, Publish, Manage, Register and Support services
+description: A history of the designs for the Find, Apply, Publish, Manage, Register, Manage school placements, Claim funding for mentor training and Support services
```

Comment:
As per my first comment. We should use an abbreviated version rather than the whole service name for 'Manage school placements' and 'Claim funding for mentor training.' This will follow the style used for the other services. For example:

```suggestion
description: A history of the designs for the Find, Apply, Publish, Manage, Register, Placements, Claim and Support services
```